deuces/setup.py

```diff
@@ -0,0 +1,24 @@
+"""
+Deuces: A pure Python poker hand evaluation library
```

Comment:
Can we make sure to add a citation and credit for all this code to the Deuces repo, if you haven't already i might have just missed it